### PR TITLE
[insteon] Binding Documentation: Updated I/O Linc Section

### PR DESCRIPTION
**Pull request by Hawkinzw**
The I/O Linc has a feature to match or invert the status of the contact with its control messages. The binding was written expecting the messages to be inverted. This is the opposite of how insteon recommends setting up the garage kit, and not described well by the original "Note" at the bottom of the section. Updated the instructions to specify the input is OFF when linking in order to invert the messages, and changed the note to better describe what happens if you don't. All based off of my own experience with the sensor: https://community.openhab.org/t/insteon-io-linc-garage-door-contact-slow-update/141469

### CODE REVIEW

File: bundles/org.openhab.binding.insteon/README.md [hunk 1]
```diff
@@ -489,7 +489,8 @@ and create a file "lock.map" in the transforms directory with these entries:
 ### I/O Linc (garage door openers)
 
 The I/O Linc devices are really two devices in one: a relay and a contact.
-Link the modem both ways, as responder and controller using the set buttons as described in the instructions.
+To control the relay, link the modem as a controller using the set buttons as described in the instructions.
+To get the status of the contact, the the modem must also be linked as a responder to the I/O Linc. The I/O linc has a feature to invert the contact or match the contact when it sends commands to any linked responders. This is based on the status of the contact when it is linked. The binding expects the contact to be inverted to work properly. Ensure the contact is OFF (status LED is dark/garage door open) when linking the the modem as a responder to the I/O Linc in order for it to function properly.
```
Comment (robnielsen) [inline bundles/org.openhab.binding.insteon/README.md:493]: Please put each sentence on it's own line.

File: bundles/org.openhab.binding.insteon/README.md [hunk 2]
```diff
@@ -517,9 +518,8 @@ To make it visible in the GUI, put this into your sitemap file:
 
 For safety reasons, only close the garage door if you have visual contact to make sure there is no obstruction! The use of automated rules for closing garage doors is dangerous.
 
-> NOTE: If the I/O Linc returns the wrong value when the device is polled (For example you open the garage door and the state correctly shows OPEN, but during polling it shows CLOSED), you probably linked the device with the PLM or hub when the door was in the wrong position.
-You need unlink and then link again with the door in the opposite position.
-Please see the Insteon I/O Linc documentation for further details.
+> NOTE: If the I/O Linc contact status appears delayed, or returns the wrong value when the sensor changes states, the contact was likely ON (status LED lit) when the modem was linked as a responder. Examples of this behavior would include: The status remaining CLOSED for up to 3 minutes after the door is opened, or the status remains OPEN for up to three minutes after the garage is opened and immedialty closed again.
```
Comment (robnielsen) [inline bundles/org.openhab.binding.insteon/README.md:521]: Please put each sentence on it's own line.

File: bundles/org.openhab.binding.insteon/README.md [hunk 3]
```diff
@@ -517,9 +518,8 @@ To make it visible in the GUI, put this into your sitemap file:
...
+To resolve this behavior the I/O Linc will need to be unlinked and then re-linked to the modem with the contact OFF (stats LED off). That would be with the door open when using the insteon garage kit.
```
Comment (robnielsen) [inline bundles/org.openhab.binding.insteon/README.md:522]: Please put each sentence on it's own line.

**Review by robnielsen — CHANGES_REQUESTED** [review]

**Comment by robnielsen** [comment]
Updates look good, other than each sentence needs to be on its own line.  You also need to sign your commit. See DCO under checks for the PR.

**Review by robnielsen — APPROVED** [review]

**Comment by Hawkinzw** [comment]
Yeah, having trouble working out signing. First time setting that up. Got it to work finally. Trying to rebase as recommended in the DCO. I'll mark ready to review again when I've got it sorted out.

**Review by lsiepel — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.insteon/README.md [hunk 4]
```diff
@@ -489,7 +489,12 @@ and create a file "lock.map" in the transforms directory with these entries:
 ### I/O Linc (garage door openers)
 
 The I/O Linc devices are really two devices in one: a relay and a contact.
-Link the modem both ways, as responder and controller using the set buttons as described in the instructions.
+To control the relay, link the modem as a controller using the set buttons as described in the instructions.
+To get the status of the contact, the the modem must also be linked as a responder to the I/O Linc.
```
Comment (jlaur) [inline bundles/org.openhab.binding.insteon/README.md:493]:
```suggestion
To get the status of the contact, the modem must also be linked as a responder to the I/O Linc.
```

File: bundles/org.openhab.binding.insteon/README.md [hunk 5]
```diff
@@ -489,7 +489,12 @@ and create a file "lock.map" in the transforms directory with these entries:
...
+The I/O linc has a feature to invert the contact or match the contact when it sends commands to any linked responders.
+This is based on the status of the contact when it is linked, and was intended for controlling other devices with the contact.
+The binding expects the contact to be inverted to work properly.
+Ensure the contact is OFF (status LED is dark/garage door open) when linking the the modem as a responder to the I/O Linc in order for it to function properly.
```
Comment (jlaur) [inline bundles/org.openhab.binding.insteon/README.md:497]:
```suggestion
Ensure the contact is OFF (status LED is dark/garage door open) when linking the modem as a responder to the I/O Linc in order for it to function properly.
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! A few typos found.

File: bundles/org.openhab.binding.insteon/README.md (hunk 5 above)
Comment (Hawkinzw) [reply bundles/org.openhab.binding.insteon/README.md:497]: good catch

**Pull request merged**